assets/pages/new-contributions-landing/contributionsLandingReducer.js

```diff
@@ -36,10 +36,16 @@ export type State = {
 // ----- Functions ----- //
 
 function createFormReducer(countryGroupId: CountryGroupId) {
+  const initialAmount: { [Contrib]: Amount } = {
+    ONE_OFF: amounts('notintest').ONE_OFF[countryGroupId][1],
```

Comment:
I wonder if we could better model the initial amount. Here you're picking the second item in every case but given work that's coming up around making the amounts (and the number of amounts) configurable through a UI, perhaps it could be better to assign an `isSelectedByDefault` or similar boolean property to the amount we want initially selected?

Reply:
Great idea! I've added that field so it can be configured by an external tool later on.